helm/oauth2-proxy/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 7.10.4
+version: 7.10.5
```

I would rate this as a new feature and not a patch. So it should be:

```suggestion
version: 7.11.0
```